modules/data.atmosphere/R/download.NARR_site.R

```diff
@@ -225,7 +225,7 @@ get_NARR_thredds <- function(start_date, end_date, lat.in, lon.in,
     PEcAn.logger::logger.info("Downloading in parallel")
     flx_df$flx <- TRUE
     sfc_df$flx <- FALSE
-    get_dfs <- dplyr::bind_rows(flx_df, sfc_df)
+    get_dfs <- dplyr::bind_rows(.data$flx_df, sfc_df)
```

Comment:
```suggestion
    get_dfs <- dplyr::bind_rows(flx_df, sfc_df)
```

Needed because (1) we want the `flx_df` defined on line 207, and (2) the arguments to `bind_rows` need to be whole dataframes rather than single columns.